docs/overview/why-saleor/extensibility.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Extensibility model
+---
+
+### API Only
+
+• **Webhooks.** You can subscribe to more than a hundred [events](/developer/extending/webhooks/overview) to receive data from Saleor.
+
+• **API extensions.** Saleor can [proxy](developer/extending/webhooks/synchronous-events/overview) calls to your service to modify behavior or resources.
+
+• **Dashboard extensions.** [Extend admin](/developer/extending/apps/overview) functionality using any front-end stack embeddable into _iframes_.
+
+• **Granular permission control.** [Granular permissions](/developer/permissions) for access tokens.
+
+• **GraphQL oriented.** Provide GraphQL queries to shape the data provided via Webhooks.
```

Comment:
Proposal:

**Powered by GraphQL.** Leverage GraphQL to tailor the data delivered through Webhooks, ensuring you receive precisely structured data payloads.

Reply:
The guide is purposefully written in plain English that avoids superfluous words